### ISSUE TITLE
Always show the geometry/project prompt when Importing .3mf file 

### ISSUE THREAD

**Issue by otomakes** [bambulab/BambuStudio#4973]
When you open a .3mf file in a new project (blank plate), all the settings and filaments are overridden with the settings from the .3mf file. If something is already on the plate, you get a prompt which asks you whether you want just the geometry or the entire project. If there is nothing on the plate, it automatically imorts the file as a project and updates all the settings. It would be really helpful if this prompt would open every time you try and open a .3mf file, as sometimes you need just the geometry from a project. 
![bambu_geometry](https://github.com/user-attachments/assets/3a60510d-7d83-46a2-a1c0-b59468724967)

**Comment by acabtp** [comment]
Agreed, it seems that with a blank plate, importing a 3mf functions to open the 3mf instead.  It should just import as expected.
